lib/src/widgets/yaru_page_indicator_layout_delegate.dart

```diff
@@ -0,0 +1,109 @@
+import 'yaru_page_indicator.dart';
+
+/// Layout delegate interface which controls a [YaruPageIndicator] items spacing.
+abstract class YaruPageIndicatorLayoutDelegate {
+  double? calculatePaneWidth({
+    required double itemsWidth,
```

Comment:
it is the width of a single item or the total width of all items?

Reply:
It is the width of all items, maybe rename it in `allItemsWidth`?

Reply:
Sounds good. I wasn't 100% sure if it was a typo or if it mean them all. :)